src/Symfony/Component/Validator/Context/LegacyExecutionContext.php

```diff
@@ -49,6 +49,8 @@ public function __construct(ValidatorInterface $validator, $root, MetadataFactor
             $translationDomain
         );
 
+        trigger_error('The LegacyExecutionContext has been deprecated and will be removed. You should use the new ExecutionContext, which now handles the functionality of this class.', E_USER_DEPRECATED);
```

Comment:
This warning is actually wrong, because it would be trigger each time the validator is used in a Symfony project.